var/spack/repos/builtin/packages/rodinia/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Rodinia(MakefilePackage, CudaPackage):
+    """Rodinia: Accelerating Compute-Intensive Applications with
+       Accelerators"""
+
+    homepage = "https://rodinia.cs.virginia.edu/doku.php"
+    url      = "http://www.cs.virginia.edu/~kw5na/lava/Rodinia/Packages/Current/rodinia_3.1.tar.bz2"
+
+    version('3.1', sha256='faebac7c11ed8f8fcf6bf2d7e85c3086fc2d11f72204d6dfc28dc5b2e8f2acfd')
+
+    depends_on('cuda')
+    depends_on('freeglut')
+    depends_on('glew')
+    depends_on('gl')
+    depends_on('glu')
+
+    conflicts('~cuda')
```

Comment:
What does this line do?

Reply:
Throws an error in `spack install rodinia ~cuda` since the package does not work without cuda.